admin-console/src/main/webapp/src/components/Wrappers/BreadCrumbLinkItem.tsx

```diff
@@ -0,0 +1,18 @@
+import { BreadcrumbItem, BreadcrumbItemProps } from '@patternfly/react-core';
+import * as React from 'react';
+
+import { OuiaComponentProps, withoutOuiaProps } from '../../utils/Ouia';
+import { LinkAdapter } from './LinkAdapter';
+
+type BreadcrumbLinkItemProps = Omit<BreadcrumbItemProps, 'component'> & OuiaComponentProps;
+
+export const BreadcrumbLinkItem: React.FunctionComponent<BreadcrumbLinkItemProps> = (props) => {
+    return (
+        <BreadcrumbItem
+            { ...withoutOuiaProps(props) }
+            component={ LinkAdapter }
+        >
+            { props.children }
+        </BreadcrumbItem>
+    );
+};
```

Comment:
Try it like this:

`withoutOuiaProps` removes the specific ouia properties from the props. But we still want the rest of the properties, so we still need to pass the `props` here.

[Ouia](https://ouia.readthedocs.io/en/latest/) is a specification used for testing, it adds some extra properties to the DOM to specify component types and make it easier for automation software to find and manage them. 

```suggestion
import { LinkAdapter } from './LinkAdapter';

type BreadcrumbLinkItemProps = Omit<BreadcrumbItemProps, 'component'>;

export const BreadcrumbLinkItem: React.FunctionComponent<BreadcrumbLinkItemProps> = (props) => {
    return (
        <BreadcrumbItem
            { ...props }
            component={ LinkAdapter }
        >
            { props.children }
        </BreadcrumbItem>
    );
};
```